.azure-pipelines/templates/jobs/standard-tests-jobs.yml

```diff
@@ -4,26 +4,24 @@ jobs:
       PYTHON_VERSION: 3.12
     strategy:
       matrix:
-        macos-py38-cover:
-          IMAGE_NAME: macOS-12
-          PYTHON_VERSION: 3.8
+        macos-py312-cover:
+          IMAGE_NAME: macOS-15
           TOXENV: cover
           # As of pip 23.1.0, builds started failing on macOS unless this flag was set.
           # See https://github.com/certbot/certbot/pull/9717#issuecomment-1610861794.
           PIP_USE_PEP517: "true"
         macos-cover:
-          IMAGE_NAME: macOS-13
+          IMAGE_NAME: macOS-15
           TOXENV: cover
           # See explanation under macos-py38-cover.
```

Comment:
we should update this comment too if we're renaming the test

i believe these two macOS tests are now identical though with the removal of setting `PYTHON_VERSION: 3.8` in the previous test. this is not at all clear, but i believe the idea here was we run macOS coverage tests once with the oldest version of python we support and once with the newest

i think my personal preference is to keep that pattern (and iif you're feeling extra motivated refactor things or add comments to make that more clear), but we could also just delete one of the two tests. i'd approve either approach

Reply:
oh, missed that, I'll change these back. still going to keep it on the updated mac version though unless there's some reason not to.